config.yaml

```diff
@@ -73,3 +74,11 @@ profiles:
       - openhim-mapping-mediator
     envFiles:
       - mpi.env
+
+  - name: reprocess
+    packages:
+      - reprocess-mediator
+      - reverse-proxy-nginx
+    envFiles:
+      - .env
+    dev: true
```

Comment:
Should not be here. The package is dependent on other packages. Put it in the recipes profiles